### ISSUE TITLE
模型细节提问

### ISSUE THREAD

**Issue by aker218** [miziha-zp/xw2020-top1#7]
大佬你好，请问一下你在卷积模型的描述的第4点：
4. 将输入特征进行分组，分别经过一个同样结构（但不共享参数）的卷积模块。这样可以使卷积核更有针对性提取各组特征内部的潜在规律，并且往往可以使用更少的卷积核得到更好的效果。
中提到的特征分组分别过同样结构的卷积模块是不是就是第7点说的用两个独立的卷积模块（同样结构，但不共享参数）分别对场景与动作进行建模呢？我看到你两个model的Input1和input2分别是8维和12维。还是说类似卷积核内部的group conv操作呢？

**Comment by miziha-zp** [comment]
这里的分组与后面建模动作与场景的目的与操作都不太一样。
从操作上讲：这里是将原始特征手动进行分组，各组经过相同结构的网络。可见，与分组卷积实际上也有区别。
从目的上讲：只允许相关性强的特征在浅层进行交互，而不是allin的在浅层交互，可以防止过拟合，特别是在数据量较少的情况下。
最后，欢迎加我QQ交流：1909856661

**Comment by aker218** [comment]
谢谢，我在您的代码部分看到的是手动将特征分成了input1的8维和input2的12维，然后input1经过对应的model部分得到分类场景的output logits ,input2经过对应的model部分得到分类动作的output logits，请问是这样的逻辑吗。

**Comment by aker218** [comment]
> 这里的分组与后面建模动作与场景的目的与操作都不太一样。
> 从操作上讲：这里是将原始特征手动进行分组，各组经过相同结构的网络。可见，与分组卷积实际上也有区别。
> 从目的上讲：只允许相关性强的特征在浅层进行交互，而不是allin的在浅层交互，可以防止过拟合，特别是在数据量较少的情况下。
> 最后，欢迎加我QQ交流：1909856661


谢谢，我在您的代码部分看到的是手动将特征分成了input1的8维和input2的12维，然后input1经过对应的model部分得到分类场景的output logits ,input2经过对应的model部分得到分类动作的output logits，请问是这样的逻辑吗。

**Comment by FizzerYu** [comment]
在参赛过程中我们发现并不是将所有的特征放在一起交互就可以获得更好的效果，因此我们将特征拆分成两组（8维和12维），相同的网络分别学习这两组特征最后再组合可以获得更好的效果（所以可以看到图上有四个分支，两个分支负责场景，两个分支负责动作，负责场景的两个分支对应两组的特征，负责动作的两个分支对应两组的特征）。

总结：

因此 我们是手动将特征分成了input1的8维和input2的12维，然后input1和input2经过同时经过model的一个分支获得分类场景的output logits，input1和input2经过同时经过model的另一个分支获得分类动作的output logits

**Comment by aker218** [comment]
> 了input1的8维和input2的12维，然后input1和input2经过同时经过model的一个

好的，又看了一遍模型图，理解了，谢谢大佬~
